p11-kit/proxy.c

```diff
@@ -67,7 +68,7 @@
 typedef struct _Mapping {
 	CK_SLOT_ID wrap_slot;
 	CK_SLOT_ID real_slot;
-	CK_FUNCTION_LIST_PTR funcs;
+	CK_FUNCTION_LIST_3_0_PTR funcs;
```

Comment:
if I change it back to CK_FUNCTION_LIST_PTR it will need to be retyped inside all 3.0 funcs, is that ok?

Reply:
I would say that should be ok. The point is to prevent accessing `CK_FUNCTION_LIST_PTR` as `CK_FUNCTION_LIST_3_0_PTR`; inside 3.0 funcs, we know what we are accessing.

Reply:
done